### DISCUSSION TITLE
Is site down?

### DISCUSSION THREAD

**Discussion by USLTD** [directvt/vtm#220] [Q&A] [answered]
@o-sdn-o and @netxs-group.
I have been wanting to ask this question for a long time:
What happened to https://vtm.netxs.online?
While SSH works, site doesn't

**Comment by o-sdn-o** [comment, marked as answer]
Unfortunately, at the moment I have no way to host SSH-live demo-server and landing page. SSH-demo server is available until the end of May 2022.

**Reply by USLTD** [reply]
Can you tell me what happened? Did host go out of service or is it something related to fees?

**Reply by o-sdn-o** [reply]
It was a big mistake to depend on a domain name, the price of which changes upwards every year. The domain name registrar charged an unreasonable lease price for the netxs.online domain name. It turned out something like a racket. I just decided not to bother.

**Reply by USLTD** [reply]
Alright. So what will be current option? GitHub pages?

**Reply by o-sdn-o** [reply]
The best landing page is the project page as it is -- https://github.com/netxs-group/vtm. The demo is best via [youtube.com](https://www.youtube.com/channel/UC1UB4zdnYUDRFH-Vh8y4EPw). There is also a twitter, reddit and telegram channel. The biggest popularity is obtained through the youtube-channel and reddit. Several times someone posted about vtm on Hacker-News, this also promoted popularity.
